Pattern: Missing use of `isEmpty`

Issue: -

## Description

Prefer checking `isEmpty` over comparing `count` to zero.

Examples of **correct** code:
```swift
var count = 0


[Int]().isEmpty


[Int]().count > 1


[Int]().count == 1


discount == 0


order.discount == 0

```
Examples of **incorrect** code:
```swift

[Int]().竊田ount == 0


[Int]().竊田ount > 0


[Int]().竊田ount != 0


竊田ount == 0

```

## Further Reading

* [SwiftLint - Empty Count](https://github.com/realm/SwiftLint/blob/master/Rules.md#empty-count)