Pattern: Leading whitespace

Issue: -

## Description

Files should not contain leading whitespace.

Examples of **correct** code:
```swift
//

```
Examples of **incorrect** code:
```swift


 //

```

## Further Reading

* [SwiftLint - Leading Whitespace](https://github.com/realm/SwiftLint/blob/master/Rules.md#leading-whitespace)