Pattern: Trailing newline

Issue: -

## Description

Files should have a single trailing newline.

Examples of **correct** code:
```swift
let a = 0

```
Examples of **incorrect** code:
```swift

let a = 0


let a = 0

```

## Further Reading

* [SwiftLint - Trailing Newline](https://github.com/realm/SwiftLint/blob/master/Rules.md#trailing-newline)